fuzzers/qemu_systemmode/src/fuzzer_classic.rs

```diff
@@ -84,9 +87,26 @@ pub fn fuzz() {
 
     let mut run_client = |state: Option<_>, mut mgr, _core_id| {
         // Initialize QEMU
-        let args: Vec<String> = env::args().collect();
+        // let args: Vec<String> = env::args().collect();
         let env: Vec<(String, String)> = env::vars().collect();
-        let qemu = Qemu::init(&args, &env).unwrap();
+        // let qemu = Qemu::init(&args, &env).unwrap();
+
+        //TODO: broken vars like ${TARGET_DIR}
+        let qemu_opt = QemuOpt::new()
+            .machine("mps2-an385".to_string())
```

Comment:
try to avoid passing string as param when it's possible to use &str instead. it makes the interface more flexible